engine/src/main/scala/cromwell/engine/workflow/tokens/JobTokenDispenserActor.scala

```diff
@@ -204,10 +205,31 @@ class JobTokenDispenserActor(override val serviceRegistryActor: ActorRef,
 
     log.info(tokenDispenserState.asJson.printWith(Printer.spaces2))
 
+    tokenEventLogger.getLimitedGroups foreach { group =>
+      log.info(s"Token Dispenser: The group $group has reached its job limit and is being rate-limited.")
+    }
+
+    tokenEventLogger.getLimitedBackends foreach { backend =>
+      log.info(s"Token Dispenser: The backend $backend is starting too many jobs. New jobs are being limited.")
+    }
+
     // Schedule the next log event:
     context.system.scheduler.scheduleOnce(someInterval) { self ! LogJobTokenAllocation(someInterval) }(context.dispatcher)
     ()
   }
+
+  /*
+  This function is not backend-aware because:
+    (1) it is used for workflow pickup decisions, and the backend is not known at that time
+    (2) the modern purpose of tokens is to manage Cromwell capacity, not backend capacity,
+          so it's desirable that a group submitting to two or more backends pause workflow
+          pickup globally when it exhausts tokens in one of the backends
+   */
+  private def getTokenExhaustedGroups: ReplyLimitedGroups = {
```

Comment:
Scala style guides tend to prefer not having `get` on the start of methods like this, eg `def tokenExhaustedGroups = ...`

EDIT: [Style guide reference](https://docs.scala-lang.org/style/naming-conventions.html#accessorsmutators)